tapscript/tx.go

```diff
@@ -220,8 +254,19 @@ func virtualTxOut(asset *asset.Asset) (*wire.TxOut, error) {
 func VirtualTx(newAsset *asset.Asset, prevAssets commitment.InputSet) (
 	*wire.MsgTx, mssmt.Tree, error) {
 
+	var (
+		txIn      *wire.TxIn
+		inputTree mssmt.Tree
+		err       error
+	)
+
 	// We'll start by mapping all inputs into a MS-SMT.
-	txIn, inputTree, err := virtualTxIn(newAsset, prevAssets)
+	switch newAsset.HasGenesisWitnessForGroup() {
```

Comment:
hehe, just `if/else`? 😛

Reply:
I thought we don't do else 😅  Can change.